### PR TITLE
Awesome Books Plain

### PR DESCRIPTION
**Pull request by Gegardus**
In this project, I manage books collection with plain JS with objects and using local storage. This app:

- creates a collection that keeps a list of books;
- adds a new book to the collection, with title and author.
- removes a book from the collection.
- Ddsplays all books saved in the collection in the top part of the page.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
```
Comment (elyor-doniyorov) [inline about.html]: - [x] Please remove `about.html` from your codebase because I think this file is not related to this project

File: script/main.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  Object.keys(removeButton).forEach((removeKey) => {
+    removeButton[removeKey].addEventListener('click', () => {
+      removeBook(removeKey);
+    });
+  }, false);
+}
+
+display();
+
+function addBook(e) {
```
Comment (elyor-doniyorov) [inline script/main.js:45]: - [x] Good job so far! But your function that adds a new book to the collection is not properly working. It is adding two books at the same time when you try for the second time and more. Let's see the below gif 👇 
20220110_164619_DURecorder_AdobeCreativeCloudExpress

Please, kindly fix it! 🙏

File: script/main.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  e.preventDefault();
+  store.push({
```
Comment (elyor-doniyorov) [inline script/main.js:47]: - [OPTIONAL] I would suggest you use array methods that doesn't mutate the array, e.g. `concat`, `filter`, `slice` instead of `push`, `pop`, `splice` and so on.

File: script/main.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+const inputAuthor = document.getElementById('author');
+
+const store = localStorage.getItem('store') ? JSON.parse(localStorage.getItem('store')) : [];
+
+function removeBook(book) {
+  const bookItems = Array.from(document.getElementsByClassName('book'));
+  Object.keys(bookItems).forEach((key) => {
+    if (book === key) {
+      bookItems[key].remove();
+      store.splice(key, 1);
```
Comment (elyor-doniyorov) [inline script/main.js:18]: - [OPTIONAL] I would suggest you use array methods that doesn't mutate the array, e.g. `concat`, `filter`, `slice` instead of `push`, `pop`, `splice` and so on.

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @Gegardus and @aeon9300 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- To add a project issue link and `instruction on how to clone and use` section in more detail. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 1 above)
Comment (Gegardus) [reply about.html]: Update Readme
Remove about.html (copypasted by mistake with README from the previous project) :)

File: script/main.js (hunk 2 above)
Comment (Gegardus) [reply script/main.js:45]: fix bag

File: script/main.js (hunk 3 above)
Comment (Gegardus) [reply script/main.js:47]: Let me, kindly, take this into account for the following projects. Thank you.

File: script/main.js (hunk 4 above)
Comment (Gegardus) [reply script/main.js:18]: Let me, kindly, take this into account for the following projects. Thank you.

**Review by Cathella — APPROVED** [review]
## 🟢 Status: Approved 🎉 💯

### Hi @Gegardus ✋🏻, 

You did a tremendous job implementing the requested changes 💪🏻.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gegardus** [comment]
Thank you, Cathella.
